inst/extdata/packages/ecospat.yaml

```diff
@@ -0,0 +1,36 @@
+name: ecospat
+title: "Spatial Ecology Miscellaneous Methods"
+version: 3.5
+author: "Olivier Broennimann, Valeria Di Cola, Blaise Petitpierre, Frank Breiner, Daniel Scherrer, Manuela D`Amen, Christophe Randin, Robin Engler, Wim Hordijk, Heidi Mod, 
+Julien Pottier, Mirko Di Febbraro, Loic Pellissier, Dorothea Pio, Ruben Garcia Mateo, Anne Dubuis, Luigi Maiorano, Achilleas Psomas, Charlotte Ndiribe, Nicolas Salamin, 
+Niklaus Zimmermann, Flavien Collart, Antoine Guisan"
+maintainer: Olivier Broennimann olivier.broennimann@unil.ch
+cran: yes
+repository: https://github.com/ecospat/ecospat
+description: "Collection of R functions and data sets for the support of spatial ecology analyses with a focus on pre, core and post modelling analyses of species distribution, 
+niche quantification and community assembly. Written by current and former members and collaborators of the ecospat group of Antoine Guisan, Department of Ecology and Evolution (DEE) 
+and Institute of Earth Surface Dynamics (IDYST), University of Lausanne, Switzerland. Read Di Cola et al. (2016) <doi:10.1111/ecog.02671> for details."
+occ_acquisition: no
+occ_cleaning: no
+data_integration: no
+env_collinearity: no
+env_process: no
+bias: yes
+study_region: no
+backg_sample: yes
```

Comment:
The function `ecospat.sample.envar` gets env raster values for occurrence points, but does not sample background values. However, the function `ecospat.rand.pseudoabsences` does, so this can stay "yes".